### PR TITLE
Change `statusBar.toggle` to receive the toggle state

### LINKED ISSUE #206: Watch and Remove watch breaks when toggled by shortcut key
**Describe the bug**
Using the keyboard to activate **Watch** and **Remove Watch** (Ctrl+Shift+8/9), if you press to any of the shortcuts it keeps changing the status bar button, but didn't change the real status.

**To Reproduce**
Steps to reproduce the behaviour:
1. Press Ctrl + shift + 8 to toggle Watch on
2. Press Ctrl + shift + 8 again
3. See error, the status bar toggle change it's text

**Expected behaviour**
Pressing the same shortcut shouldn't change the toggle text

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Windows
 - Extension Version 2.2.0
 - VSCode Version 1.30.2

### PR DESCRIPTION
**Pull request by felipecaputo**
Fix #206

### CODE REVIEW

File: src/extension/statusbartoggler.ts [hunk 1]
```diff
@@ -19,11 +19,15 @@ export class StatusBarToggler implements Disposable {
...
+        return this.statusBarItem.text;
+    }
+
     /**
      * Toggles the status bar item from watch to remove and vice versa
      */
-    public toggle() {
-        if (this.statusBarItem.command === StatusBarToggler.watchCommand) {
+    public toggle(active: boolean) {
+        if (active) {
```
Comment (ryanluker) [inline src/extension/statusbartoggler.ts]: ahh this is much nicer then the command name comparison!

File: src/extension/statusbartoggler.ts [hunk 2]
```diff
@@ -19,11 +19,15 @@ export class StatusBarToggler implements Disposable {
         if (this.configStore.showStatusBarToggler) { this.statusBarItem.show(); }
     }
 
+    public get statusText() {
```
Comment (ryanluker) [inline src/extension/statusbartoggler.ts]: nice use of the es6 object getter.

File: test/extension/statusbartoggler.test.ts [hunk 3]
```diff
@@ -1,13 +1,34 @@
...
+    test("Should not toggle twice showStatusBarToggler command and message @unit", () => {
+        const statusBarToggler = new StatusBarToggler(fakeConfig);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Watch");
+        statusBarToggler.toggle(true);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Remove Watch");
+        statusBarToggler.toggle(true);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Remove Watch");
+    });
+
+    test("Should toggle showStatusBarToggler command and message back to \"Wath\" @unit", () => {
```
Comment (ryanluker) [inline test/extension/statusbartoggler.test.ts]: you probably meant `\"Watch"`

Comment (felipecaputo) [reply test/extension/statusbartoggler.test.ts]:
```suggestion
    test("Should toggle showStatusBarToggler command and message back to \"Watch\" @unit", () => {
```

Thank you for the quick response. GitHub suggested me to use this tool that I can edit the code in the comment, If it doesn't work I'll update the MR as soon as I get home.

And thanks for the great extension you did! o/

**Pull request merged**